clients/java/src/test/java/io/zeebe/client/job/ExponentialBackoffTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.job;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.zeebe.client.api.worker.BackoffSupplier;
+import io.zeebe.client.impl.worker.ExponentialBackoff;
+import io.zeebe.client.impl.worker.ExponentialBackoffBuilderImpl;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.LongStream;
+import org.junit.Test;
+
+public final class ExponentialBackoffTest {
+  @Test
+  public void plotIt() {
```

Comment:
Is this a test? I think I added that to check the distribution, we can probably remove it :sweat_smile: